components/brave_shields/resources/panel/components/main-panel/index.tsx

```diff
@@ -89,6 +89,16 @@ function MainPanel () {
     )
   }
 
+  const renderTotalCount = () => {
```

Comment:
nit: since this is only called once I don't see why it should be a function. It can just be a variable, right?

Reply:
this make sense